### PR TITLE
HCAP-1106 | Add date to last engaged by column

### PR DESCRIPTION
**Pull request by taraepp**
- date added
- format according to MMM DD, YYYY
- made classes for the cell text in participant table
- added that darker grey colour to theme palette
- did not change the sorting (sorting by date not desired, already sorting by name)

Could have instead used Typography instead of classes? Not sure if we like div but it gets the job done.

Updated:
- investigated using Typography instead: v4 of MUI does not support custom variants, the feature was added in v5. None of our existing variants matched the design (all bold variants were much larger or the button variant has a text-transform). Did not want to modify variants used elsewhere.
- changed div to Box
- renamed that class to Alva's suggestion
- did not follow the design _exactly_ as it had the staff name text larger than the date. The design also had other larger, bold text (column headers) that the implementation currently lacks. It looked out of place.

### CODE REVIEW

File: client/src/pages/private/ParticipantTable.js [hunk 1]
```diff
@@ -401,6 +417,13 @@ const ParticipantTable = () => {
         );
       case 'postHireStatuses':
         return getGraduationStatus(row[columnId] || []);
+      case 'engagedBy':
+        return (
+          <>
+            <div className={classes.cellTextStrong}>{row.engagedBy}</div>
```
Comment (alva-fresh) [inline client/src/pages/private/ParticipantTable.js:423]: Can we replace _div_ either with `Box` or `Typography`? I feel like for `Typography` we already have some predefined styles, so there's no need to introduce `cellTextStrong`

File: client/src/pages/private/ParticipantTable.js [hunk 2]
```diff
@@ -190,6 +196,16 @@ const ParticipantTable = () => {
   const isAdmin = isMoH || isSuperUser;
   const isEmployer = roles.includes('health_authority') || roles.includes('employer');
 
+  const useStyles = makeStyles((theme) => ({
+    cellTextStrong: {
+      fontWeight: 'bold',
+    },
+    cellTextWeak: {
```
Comment (alva-fresh) [inline client/src/pages/private/ParticipantTable.js:203]: nit: maybe `cellTextGray` sounds beeter?

Comment (taraepp) [reply client/src/pages/private/ParticipantTable.js:203]: Much better! Thanks for the suggestion, I didn't like like what I had either.

File: client/src/pages/private/ParticipantTable.js (hunk 1 above)
Comment (taraepp) [reply client/src/pages/private/ParticipantTable.js:423]: My only concern with Typography is that it won't introduce a line break, but I'll do some experimenting with Typography and Box to confirm and find something that works- I'll get rid of the div.

**Review by fwpushan — COMMENTED** [review]
1. I don't see any column config here
2. what about sort logic?

**Review by alva-fresh — APPROVED** [review]
Looks good!

**Review by fwpushan — APPROVED** [review]

**Pull request merged**